ui/src/FindBar.tsx

```diff
@@ -131,22 +219,26 @@ export class FindBar extends React.PureComponent<{}, FindBarState> {
       return null;
     }
 
-    if (matchCount > MATCH_COUNT_LIMIT) {
+    if ((matchCount || 0) > MATCH_COUNT_LIMIT) {
       return `More than ${MATCH_COUNT_LIMIT} matches`;
     }
 
     return `${currentMatch} of ${matchCount} matches`;
   }
 
   render() {
+    if (this.state.mode === 'hidden') {
+      return <div></div>
```

Comment:
Why not return null or undefined?

Reply:
I can change this to null if you want :), I believe returning undefined from render is not allowed